### ISSUE TITLE
Резкая смерть всех игроков стоящих на земле. #BUG

### ISSUE THREAD

**Issue by ZhenyaMartynov** [Sokol-Games-GmbH/PixelCombats.PublicApi#134]
Если в режиме будет включен полет, и игрок в команде flush улетит тупо в низ далеко, например -9999999. - то все игроки стоящие на блоках умрут в этот момент, или через некоторое время. Могу продемонстрировать вам прямо на сервере

Выдайте тестера за баг
ID:
C002224F3666744D

**Comment by forwork10** [comment]
этот баг известен давно

**Comment by ZhenyaMartynov** [comment]
Ахахахах блять, ты долбаеб спт? Нет, когда ты кидаешь хуйню про то что снаряд летит не туда, это новое и актуальное! А это нет? Так никто не кидал этого НЕ РАЗУ, а про рпг там постов 20 в альфе, ты просто гений!

**Comment by forwork10** [comment]
Так, я понимаю что я вас взбесил.. Но усвойте, что баг, предложенный вами сверху, никак не мешает игровому процессу, ведь там где есть полёт - игроки его и используют(не умирают).
А вот баг за взрывчаткой мешает во всех случаях...

**Comment by ZhenyaMartynov** [comment]
Ауу) как твой высер поменял суть моего сообщения? То что ты пернул чё то по поводу что уже знают, а сам высрал хуйню про рпг

**Comment by forwork10** [comment]
? вы не поняли суть моего сообщения? или что...

**Comment by ZhenyaMartynov** [comment]
Господи, какой же ты идиот!

**Comment by ZhenyaMartynov** [comment]
Я не буду дальше спорить с долбаебом который не прав даже, не буду время на хуя из инета тратить

**Comment by forwork10** [comment]
Я задал вопрос, вы не ответили... И зачем вы пишите что я якобы не прав? Вы кому тут это доказать пытаетесь? Это выглядит как вы от чувства неуверенности в себе или зависти пытаетесь написать то чего на самом деле нет, тоесть самоутверждение, но оно ложное...

**Comment by kkohno** [comment]
> Если в режиме будет включен полет, и игрок в команде flush улетит тупо в низ далеко, например -9999999. - то все игроки стоящие на блоках умрут в этот момент, или через некоторое время. Могу продемонстрировать вам прямо на сервере
> 
> Выдайте тестера за баг ID: C002224F3666744D

можно подробнее, со всеми командами, что пишете , что вызываете, чтобы повторить баг

**Comment by forwork10** [comment]
> > Если в режиме будет включен полет, и игрок в команде flush улетит тупо в низ далеко, например -9999999. - то все игроки стоящие на блоках умрут в этот момент, или через некоторое время. Могу продемонстрировать вам прямо на сервере
> > 
> > Выдайте тестера за баг ID: C002224F3666744D
> 
> можно подробнее, со всеми командами, что пишете , что вызываете, чтобы повторить баг

заходите в любую комнату, вводите команду flyh с высотой полёта -999 999 999 999 и все игроки, у которых полёт выключен, не могут прыгать и умирают, так как игра считает что они падают.

**Comment by kkohno** [comment]
ааа, понял))

**Comment by ZhenyaMartynov** [comment]
Тестер выдайте за баг

**Comment by forwork10** [comment]
> Тестер выдайте за баг 

да кста, айдт 2F665AF97FA6F0EF

**Comment by ZhenyaMartynov** [comment]
> > Тестер выдайте за баг 
> 
> да кста, айди 2F665AF97FA6F0EF 

Нет)) я описал баг выше первее, и тема моя

Айди
C002224F3666744D

**Comment by ZhenyaMartynov** [comment]
> > > Тестер выдайте за баг 
> > 
> > да кста, айди 2F665AF97FA6F0EF 
> 
> Нет)) я описал баг выше первее, и тема моя
> 
> Айди
> C002224F3666744D

![Screenshot_2023-10-22-14-43-14-573_com.github.android.jpg](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/146564992/f3845d6b-d69a-4ded-8acc-a0019edee441)

**Comment by forwork10** [comment]
> > > Тестер выдайте за баг 
> > 
> > да кста, айди 2F665AF97FA6F0EF 
> 
> Нет)) я описал баг выше первее, и тема моя
> 
> Айди
> C002224F3666744D

твоё говно описание никто не понял) а вот моё - да) просто почитай выше.

2F665AF97FA6F0EF

**Comment by ZhenyaMartynov** [comment]
> > > > Тестер выдайте за баг 
> > > 
> > > да кста, айди 2F665AF97FA6F0EF 
> > 
> > Нет)) я описал баг выше первее, и тема моя
> > 
> > Айди
> > C002224F3666744D
> 
> твоё говно описание никто не понял) а вот моё - да) просто почитай выше.
> 
> **2F665AF97FA6F0EF** 

Долбаеб) само критика? Сам себя говном полить решил?
Ты переписал мой текст лол, суть та же

**Comment by forwork10** [comment]
> > > > > Тестер выдайте за баг 
> > > > 
> > > > да кста, айди 2F665AF97FA6F0EF 
> > > 
> > > Нет)) я описал баг выше первее, и тема моя
> > > 
> > > Айди
> > > C002224F3666744D
> > 
> > твоё говно описание никто не понял) а вот моё - да) просто почитай выше.
> > 
> > **2F665AF97FA6F0EF** 
> 
> Долбаеб) само критика? Сам себя говном полить решил?
> Ты переписал мой текст лол, суть та же 

переписал текст(дристня непонятная), и стало понятнее.... Молчал бы ты лучше.. ты со своими навыками общения в 12 лет даже не сможешь объяснить ничего толком..

**Comment by ZhenyaMartynov** [comment]
Спт ты просто клоун))) пытаешся уже на моих бегах накопится, гений! Я умнее и старше тебя) ты 13 летняя хуйня, которая нечего не добился. И щас пытаешься МНЕ что то сделать от безысходности, потому что ты понимаешь что я сильнее, я умнее, а ты просто 0, ты нулевка, ты нечего не можешь сделать

Я перекрыл тебе накрутку, слил тебя много раз, опозорил публично,Я ДОКАЗАЛ ЧТО Я СИЛЬНЕЕ УЖЕ ВСЕМ. Я понимаю что ты зол на то что я стал сильным

**Comment by forwork10** [comment]
> Спт ты просто клоун))) пытаешся уже на моих бегах накопится, гений! Я умнее и старше тебя) ты 13 летняя хуйня, которая нечего не добился. И щас пытаешься МНЕ что то сделать от безысходности, потому что ты понимаешь что я сильнее, я умнее, а ты просто 0, ты нулевка, ты нечего не можешь сделать
> 
> Я перекрыл тебе накрутку, слил тебя много раз, опозорил публично,Я ДОКАЗАЛ ЧТО Я СИЛЬНЕЕ УЖЕ ВСЕМ. Я понимаю что ты зол на то что я стал сильным 

ну вот ты опять себя утешаешь... говоришь что СИЛЬНЕЕ МЕНЯ и тд, а это потом что ты сам так не думаешь)) но хочешь чтобы другие так думали)) 

а ещё вижу твоя личная жизнь становится все хуже и хуже, ты так жаждишь внимания)
